### PR TITLE
[zh] sync kubeadm-reconfigure configure-service-account kubectl-node-debug custom-resource-definitions

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -421,7 +421,7 @@ token:          ...
 
 {{< note >}}
 <!--
-The content of `token` is elided here.
+The content of `token` is omitted here.
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md:424]: It's slightly different. Do we need to change Chinese to `省略了`?

Comment (tengqm) [reply content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md:424]: should be fine.

File: content/zh-cn/docs/tasks/debug/debug-cluster/kubectl-node-debug.md [hunk 2]
```diff
@@ -124,15 +124,18 @@ When creating a debugging session on a Node, keep in mind that:
...
 -->
 当在节点上创建一个调试会话时，需谨记：
 
 * `kubectl debug` 根据节点的名称自动生成新 Pod 的名称。
 * 节点的根文件系统将被挂载在 `/host`。
 * 尽管容器运行在主机 IPC、Network 和 PID 名字空间中，但 Pod 没有特权。
   这意味着读取某些进程信息可能会失败，这是因为访问这些信息仅限于超级用户 (superuser)。
-  例如，`chroot /host` 将失败。如果你需要一个有特权的 Pod，请手动创建。
+  例如，`chroot /host` 将失败。如果你需要一个有特权的 Pod，请手动创建或使用 `--profile=sysadmin` 标志。
+* 通过应用[调试配置文件](/zh-cn/docs/tasks/debug/debug-application/debug-running-pod/#debugging-profiles)，
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/debug/debug-cluster/kubectl-node-debug.md:137]:
```suggestion
* 通过应用[调试配置](/zh-cn/docs/tasks/debug/debug-application/debug-running-pod/#debugging-profiles)，
```

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Pull request merged**